### PR TITLE
New clown donator items

### PR DESCRIPTION
**Pull request by Crumpaloo**
## About The Pull Request

This PR includes 4 new aesthetic related icons following a clown theme as donator items for  BoisterousBeebz and  Aether217:

- Bubbly Clown Dress (only accessible to the donors as specified above)
- Pink Clown Shoes (job restricted only to clown roles)
- Flushed Cheeks (head marking)
- Nose Blemish (head marking)

image

## Changelog

:cl:
imageadd: New clown dress, clown shoes, clown related markings
:cl:

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -496,3 +496,9 @@
 	donator_only = TRUE
 
 //Chunko Fops were donated by SlippyJoe, who requested they are usable by everyone
+
+/datum/loadout_item/under/jumpsuit/bubblyclown
+	name = "Bubbly Clown Dress"
+	item_path = /obj/item/clothing/under/bubblyclown/skirt
+	restricted_roles = list(JOB_CLOWN)
+	ckeywhitelist = list("boisterousbeebz", "aether217",)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	ckeywhitelist = list("boisterousbeebz", "aether217")
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 2]
```diff
@@ -496,3 +496,9 @@
 	donator_only = TRUE
 
 //Chunko Fops were donated by SlippyJoe, who requested they are usable by everyone
+
+/datum/loadout_item/under/jumpsuit/bubblyclown
+	name = "Bubbly Clown Dress"
+	item_path = /obj/item/clothing/under/bubblyclown/skirt
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
/datum/loadout_item/under/jumpsuit/bubbly_clown
	name = "Bubbly Clown Dress"
	item_path = /obj/item/clothing/under/bubbly_clown/skirt
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm [hunk 3]
```diff
@@ -61,6 +61,19 @@
...
+	name = "Flushed Cheeks"
+	icon_state = "flushed_cheeks"
+	default_color = "#CCCCCC"
+	affected_bodyparts = HEAD
+
+/datum/body_marking/other/noseblemish
+	name = "Nose Blemish"
+	icon_state = "nose_blemish"
+	default_color = "#CCCCCC"
+	affected_bodyparts = HEAD
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm]:
```suggestion
/datum/body_marking/other/flushed_cheeks
	name = "Flushed Cheeks"
	icon_state = "flushed_cheeks"
	default_color = "#CCCCCC"
	affected_bodyparts = HEAD

/datum/body_marking/other/nose_blemish
	name = "Nose Blemish"
	icon_state = "nose_blemish"
	default_color = "#CCCCCC"
	affected_bodyparts = HEAD
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 4]
```diff
@@ -1421,3 +1421,15 @@
 /obj/item/clothing/accessory/hypno_watch/examine()
 	. = ..()
 	. += span_boldwarning("Who knows what it could be used for?")
+
+// Donation reward for BoisterousBeebz
+
+/obj/item/clothing/under/bubblyclown/skirt
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
/obj/item/clothing/under/bubbly_clown/skirt
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 5]
```diff
@@ -1421,3 +1421,15 @@
...
+	name = "Bubbly Clown Dress"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: Item names are always in lower case unless it's proper names.
```suggestion
	name = "bubbly clown dress"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Only small things, haven't taken a look at the sprites themselves yet.

File: code/modules/clothing/shoes/clown.dm [hunk 6]
```diff
@@ -46,3 +46,9 @@
 	name = "jester shoes"
 	desc = "A court jester's shoes, updated with modern squeaking technology."
 	icon_state = "jester_shoes"
+
+/obj/item/clothing/shoes/clown_shoes/pink
+	name = "Pink Clown Shoes"
+	desc = "A particularly pink pair of punny shoes."
+	icon_state = "pink_clown_shoes"
+	inhand_icon_state = "pink_clown_shoes"
```
Comment (GoldenAlpharex) [inline code/modules/clothing/shoes/clown.dm]: This should be moved to a modular file.
Also, the name should be `pink clown shoes`

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
There shouldn't be icon changes to the following files:
`icons/mob/clothing/feet.dmi`
`icons/obj/clothing/shoes.dmi`
`modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi`

**Comment by dawsonkeyes** [comment]
`when the clussy hits`

**Pull request merged**